maps/include/maps/G3SkyMap.h

```diff
@@ -354,19 +419,19 @@ class G3SkyMapWithWeights : public G3FrameObject {
 	G3SkyMapWeightsPtr RemoveWeights();
 	void ApplyWeights(G3SkyMapWeightsPtr weights);
 
-	StokesVector get_interp_value(double alpha, double delta) const;
-
 	boost::shared_ptr<G3SkyMapWithWeights> Rebin(size_t scale) const;
 
+	void Compress(bool zero_nans = false);
```

Comment:
I'm not sure "Compress" is the right name for what this does, since it is not what most people think of by "compression". Maybe "Compact"?

Reply:
This and others fixed in 5546cbf